api/query.py

```diff
@@ -253,3 +267,31 @@ def get_invalid_taint_models(
                 )
             )
     return errors
+
+
+def types(pyre_connection: PyreConnection, modules: Iterable[str]) -> List[Types]:
```

Comment:
Please add tests for this function. Tests are here:
https://github.com/facebook/pyre-check/blob/11ab586bd77f97c5778d89a9338f178203569ee1/api/tests/query_test.py